docs/governance/community-groups.md

```diff
@@ -22,6 +22,45 @@ Non-technical issues or high-level design decisions may be escalated to the [SC]
 
 Technical issues should be resolved by the relevant community group. Failure to resolve technical issues may be escalated to the [SC] for guidance on finding a resolution.
 
+## Working Groups
+
+The CCC project is split into several separate, but related, working groups. Each working group focuses on a specific sub-area of the CCC problem space.
+
+### Working Groups Structure
+
+Working groups are structured with distinct roles to ensure effective operation and governance
+
+1. Working Group Lead: The lead orchestrates the group’s activities, liaises with the broader project community, and represents the group in larger project discussions. As the primary contact, the lead ensures smooth communication and progress.
+2. Maintainers: These are project maintainers deeply invested in the group’s focus areas. While a maintainer may also serve as the lead, it is not a requirement.
+3. PR Approvers: Selected members are entrusted with the authority to review, approve, and merge pull requests (PRs), maintaining the quality and integrity of contributions.
```

Comment:
I think we should remove the PR merge responsibility from the approver. PR approver should only be responsible to approve/reject a PR. Maintainer may be a good person to be responsible for merging a PR.
I also think we should have an approval count before a PR is considered ready for merging.

Reply:
Good point, I have made the suggested changes.